pages/Cards.vue

```diff
@@ -1,70 +1,86 @@
 <template>
-  <div>
-    <v-container class="card-container">
-      <v-layout row wrap>
-        <v-flex
-          xs1
-          sm2
-          md3
-          lg4
-          v-for="article in articles.slice(initial, final)"
-          v-bind:key="article.title"
-        >
-          <v-card flat class="text-center ma-3"
-            ><a class="text-link title" target="_blank" :href="article.url">{{
-              article.title
-            }}</a>
+    <div>
+        <v-container class="card-container">
+            <v-layout row wrap>
+                <v-flex
+                    xs1
+                    sm2
+                    md3
+                    lg4
+                    v-for="article in articles.slice(initial, final)"
+                    :key="article.title"
+                >
+                    <v-card flat class="text-center ma-3"
+                        ><a
+                            class="text-link title"
+                            target="_blank"
+                            :href="article.url"
+                            >{{ article.title }}</a
+                        >
 
-            <v-img :src="article.urlToImage"></v-img>
-            <div class="subheading">{{ article.url }}</div>
-            <div class="grey--text">20th January 2023</div>
-            <a class="text-link" target="_blank" :href="article.url">
-              <v-btn>View Details</v-btn>
-            </a>
-          </v-card>
-        </v-flex>
-      </v-layout>
+                        <v-img :src="article.urlToImage"></v-img>
+                        <div class="sub-heading">{{ article.url }}</div>
+                        <div class="grey--text">
+                            {{
+                                new Date(
+                                    article.publishedAt
+                                ).toLocaleDateString("en-in", {
+                                    day: "numeric",
+                                    month: "long",
+                                    year: "numeric",
+                                })
+                            }}
+                        </div>
+                        <a
+                            class="text-link"
+                            target="_blank"
+                            :href="article.url"
+                        >
+                            <v-btn>View Details</v-btn>
+                        </a>
```

Comment:
```suggestion 
<v-btn :href="article.url" target="_blank">View Details</v-btn>
```
and why are you applying class on anchor tag not on v-btn?